qa/pull-tester/rpc-tests.sh

```diff
@@ -213,20 +213,14 @@ fi
 if [ ! -z "$SPLIT" ]; then
   chunks="${SPLIT%*:*}"
   chunk="${SPLIT#*:}"
-  chunkSize="$((${#testScripts[@]}/${chunks}))"
-  start=0
-  for (( i = 1; i <= $chunks; i++ )); do
-    if [ $i -eq $chunk ]; then
-      break
-    fi
-    start=$(($start+$chunkSize))
-  done
-  if [ $chunks -eq $chunk ]; then
-    testScripts=( "${testScripts[@]:$start}" )
-  else
-    testScripts=( "${testScripts[@]:$start:$chunkSize}" )
-  fi
+else
+  chunks='1'
+  chunk='1'
 fi
+# Temporary replace the array delimiter from ' ' to '|'
+originalIFS="$IFS"; IFS='|'
+read -a testScripts <<< $(${BUILDDIR}/qa/pull-tester/loadbalancer.py $chunks $chunk ${testScripts[@]})
+IFS="$originalIFS"
```

Comment:
Just throwing an idea here, feel free to ignore it. With something like the following, you should have a much easier to handle list of arguments passed to your Python script. You might be able to simplify things a bit, especially lines 15-29 of `loadbalancer.py`:
```suggestion
read -a testScripts <<< $(${BUILDDIR}/qa/pull-tester/loadbalancer.py $chunks $chunk $(printf '"%s" ' "${testScripts[@]}"))
```

Reply:
Thanks! I integrated this change and added a simplified scheme for passing parameter back and forth the Python script.

Reply:
Awesome, thank you!